### PR TITLE
[doc] PR for Documentation for support bundle

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-2995

Preview: https://deploy-preview-12453--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/troubleshoot/universe-issues/#use-support-bundles

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md [hunk 1]
```diff
@@ -302,4 +302,54 @@ If you disable **Assign Public IP** during the universe creation, the process ma
...
+- G-flag configuration files containing the G-flags set on the universe.
+- Instance files that contain the metadata information from the Master and TServer.
+- Consensus meta files containing consensus metadata information from the Master and TServer.
+- Tablet meta files containing the tablet metadata from the Master and TServer.
+
+The diagnostic information can be analyzed locally or the bundle can be forwarded to the Yugabyte Support team. 
+
+You can create a support bundle as follows:
+
+- Open the universe that nees to be diagnosed and click **Actions > Support Bundles**.
```
Comment (Sahith02) [inline docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:324]: Spelling mistake: needs

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md [hunk 2]
```diff
@@ -302,4 +302,54 @@ If you disable **Assign Public IP** during the universe creation, the process ma
...
+
+  <br><br>
+
+  ![Create support bundle](/images/yp/support-bundle-2.png)
+
+  <br><br>
+
+- Click **Create Bundle**.
+
+  YugabyteDB Anywhere starts collecting files from all the nodes in the cluster into an archive. Note that this process might take a few minutes. When finished, the bundle's status is displayed as **Ready**, as per the followingn illustration:
```
Comment (Sahith02) [inline docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:344]: Spelling mistake: following

**Review by Sahith02 — CHANGES_REQUESTED** [review]
All looks good, except for minor spelling mistakes commented. 
Btw we should probably mention somewhere that this feature is only available from 2.13.2 release onwards.

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:324]: fixed

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:344]: fixed

**Comment by Sahith02** [comment]
@lizayugabyte Can you also add that this feature is only available from 2.13.2 release? Thanks

**Comment by lizayugabyte** [comment]
@Sahith02 The doc will be published in 2.13.2. There's no need to state the release in the text.

**Pull request merged**